### ISSUE TITLE
Linux, can't download Blender

### ISSUE THREAD

**Issue by Nesakko** [DotBow/Blender-Launcher#5]
**Describe the bug**

The launcher launches very well, but doesn't download anything, whether I launch as sudo or not

**To Reproduce**

Launch the application
Click download button

**Expected behavior**

Log from BL.log

```
2020-05-26 17:55:30,815 - __main__ - ERROR - Linux
Traceback (most recent call last):
  File "urllib/request.py", line 1326, in do_open
  File "http/client.py", line 1240, in request
  File "http/client.py", line 1286, in _send_request
  File "http/client.py", line 1235, in endheaders
  File "http/client.py", line 1006, in _send_output
  File "http/client.py", line 946, in send
  File "http/client.py", line 1409, in connect
  File "ssl.py", line 500, in wrap_socket
  File "ssl.py", line 1040, in _create
  File "ssl.py", line 1309, in do_handshake
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1108)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "threads/downloader.py", line 27, in run
  File "urllib/request.py", line 222, in urlopen
  File "urllib/request.py", line 525, in open
  File "urllib/request.py", line 542, in _open
  File "urllib/request.py", line 502, in _call_chain
  File "urllib/request.py", line 1369, in https_open
  File "urllib/request.py", line 1329, in do_open
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1108)>

```

**System :**
 - Operating system: Manjaro
 - Blender Launcher version: v1.0.0

**Comment by DotBow** [comment]
Hi!
Thanks for report, working on it.

**Comment by DotBow** [comment]
@Nesakko please, check if this build fixes issue for you
https://drive.google.com/open?id=1sA6aR8xhNa5xpp8Okint0nVMN3k3s959

**Comment by Nesakko** [comment]
Thanks. It works perfectly

**Comment by DotBow** [comment]
Amazing! Fixed 3117fa6e4ab379b2ecbd5ddfaaf0b1085506ebe1
